libnavui-maps/src/test/java/com/mapbox/navigation/ui/maps/internal/route/line/MapboxRouteLineUtilsTest.kt

```diff
@@ -1755,11 +1756,11 @@ class MapboxRouteLineUtilsTest {
 
     @Test
     fun buildWayPointFeatureCollection() {
-        val route = getMultilegRoute()
+        val route = getMultilegRoute().toNavigationRoute()
 
         val result = MapboxRouteLineUtils.buildWayPointFeatureCollection(route)
 
-        assertEquals(4, result.features()!!.size)
+        assertEquals(3, result.features()!!.size)
```

Comment:
Did the fixture change? 🤔

Reply:
There was a bug in the route line implementation where an intermediate waypoint geometry was added to the list twice (you can see that in the old test where the point is duplicated). It wasn't visible since the icons where just shown on top of each other. This PRs fixes the bug.

cc @cafesilencio for visibility